src/components/home.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+const Home = () => (
+  <div className="home-container">
+    <h1 className="home-heading">Welcome to our page!</h1>
+    <p className="home-para">We have made this web application to give you access to a magic calculator that you can use anytime, anywhere.</p>
+    <p className="home-para">You will use all possible basic Math operations such as Addition, Subtraction, Multiplication, Division, plus some advanced operations such as Percentage, Exponentiation and more...</p>
+    <p className="home-para">Explore them on the links above</p>
```

Comment:
- [ ] Optional suggestion

   - Kindly, keep one `<p>` tag, with all the text, Imagine once you have more than 100 lines with the `<p>` tag. How would manage it?